materials/models/player/boss/uber_ranger/uberranger_backpack.vmt

```diff
@@ -0,0 +1,67 @@
+"VertexLitGeneric"
+{
+	"$basetexture"	"models/player/boss/uber_ranger/uberranger_backpack"
+	"$basemapalphaphongmask" "1"
+		"$detail" "effects/tiledfire/fireLayeredSlowTiled512.vtf"	
+	"$detailscale" "2"
+	"$detailblendfactor" .01
+	"$detailblendmode" 6
+
+	"$blendtintbybasealpha" "1" 
+ 	"$blendtintcoloroverbase" "0"  
+
+	"$colortint_base" "{230 230 230}"  
+	"$colortint_tmp" "[0 0 0]"
+
+// 	"$bbqamount" "1"	
+// 	"$two" "2.35"
+// 	"$one" "1"
+// 	"$absburnminusone" "1"
+// 	"$burnminusone" "1"
+// 	"$finalburnlevel" "1"
+// 	"$burnlevel" "0"
+
+
+	"$phong" "1"
+	"$phongexponent" "20"
+	"$phongboost" ".3"	
+	"$lightwarptexture" "models/player/pyro/pyro_lightwarp"
+	"$phongfresnelranges"	"[.3 1 8]"
+	"$halflambert" "0"
+
+	// Rim lighting parameters
+	"$rimlight" "1"												// To enable rim lighting (requires phong)
+	"$rimlightexponent" "4"										// Exponent for phong component of rim lighting	
+	"$rimlightboost" "2"										// Boost for ambient cube component of rim lighting
+
+	// Cloaking
+	"$cloakPassEnabled" "1"
+
+	"$yellow" "0"
+
+	"Proxies"
+	{
+		"spy_invis"
+		{
+		}
```

Comment:
Is this needed?

Reply:
The .vmts are all unchanged from their original textures besides adding paintability support (so all inconsistencies you see are to be blamed on valve)